dead.py

```diff
@@ -26,7 +26,9 @@
 # https://github.com/python/typed_ast/blob/55420396/ast27/Parser/tokenizer.c#L102-L104
 TYPE_COMMENT_RE = re.compile(r'^#\s*type:\s*')
 # https://github.com/python/typed_ast/blob/55420396/ast27/Parser/tokenizer.c#L1400
```

Comment:
this comment should be updated to refer to the new parsing -- at the time this was written the regex matched the typed-ast parser